tests/repoFindByDate.py

```diff
@@ -158,7 +158,7 @@ def map_cfg(self, dataId, write):
         lookups = self.storage.lookup(lookupProperties='date', reference=None,
                                      dataId=dataId, template=template)
         lookups.sort()
-        if len(lookups) is not 0:
+        if len(lookups):
```

Comment:
IMO 'is not 0' is more expressive of intent; I'm not interested if the value is True (or False), but that the length of the list is > 0. 
I suppose there's something contrary to my perspective in python coding standards, but that's my $0.02.